src/targets/maven.ts

```diff
@@ -269,6 +275,10 @@ export class MavenTarget extends BaseTarget {
       `Extracting ${artifact.filename} to ${downloadedPkgPath}...`
     );
     await extractZipArchive(downloadedPkgPath, dir);
+    // All artifacts downloaded from GitHub are ZIP files.
+    const pkgName = basename(artifact.filename, '.zip');
+    const distDir = join(dir, pkgName);
+    await this.uploadDistribution(distDir);
```

Comment:
Still not following here. We already unpacked the `zip` archive at this point, right? Why do we still need that file or its filename?

Reply:
In order to publish a distribution, one of the files we need to find is the release file; a `.aar` for Android releases and a `.jar` for the others. Instead of traversing the whole directory, we can identify the file by doing some string replacement in the distribution name, since names are pretty similar. We need the name of the distribution for that, and the filename of the `zip` is the distribution directory.

Reply:
This sounds like a great piece of comment that can live inside the code.